src/app/players/players.component.ts

```diff
@@ -143,7 +149,10 @@ export class PlayersComponent implements OnInit, OnDestroy {
 
   openNewPlayerForm() {
     this.isNewPlayerFormGroupVisible = true;
-    setTimeout(() => this.nameNewPlayer.focus(), 0);
+    setTimeout(
+      () => this.renderer.selectRootElement('#nameNewPlayer').focus(),
```

Comment:
I'm really not convinced that's needed :think:

Reply:
Removed.